content/en/synthetics/browser_tests/actions.md

```diff
@@ -202,12 +202,13 @@ To create a variable, first give it a name then define its value from:
 
 Create a variable by defining its value from one of the below available builtins: 
```

Comment:
@MargotLepizzera It looks like this file isn't populating in the nav menu on the docs site. If you click Synthetic Monitoring in the menu, it only shows: Steps, Test Results, Advanced Options for Steps. Is this intentional? If so, I'll leave as-is. If we want it in the main nav, I can add.